Tests/headers/libIARM.h

```diff
@@ -2,6 +2,7 @@
 
 #define IARM_BUS_NAME "com.comcast.rdk.iarm.bus"
 #define IARM_MAX_NAME_LEN 64
+#define IARM_BUS_COMMON_API_SysModeChange "SysModeChange"
```

Comment:
looks like it was added where it just works. If the definition is not from libIARM.h, it should not be here. Same applies to other changes in this pull request

Reply:
synchronized the changes with the original files in iarmbus and powermanager: moved some changes to libIBusDaemon.h and included the file accordingly